tests/test_runtime.py

```diff
@@ -43,7 +44,8 @@
 class TestAsyncRuntime:
     """Test async runtime."""
 
-    RUNTIME: Type[BaseRuntime] = AsyncRuntime
+    # set a copy to prevent lasting state changes via class attributes
+    RUNTIME: Type[BaseRuntime] = copy_class(AsyncRuntime)
```

Comment:
`test_supervisor_not_started` failure still occurs on ubuntu 3.8 and 3.10 in after merge of #414 - commit [8152d4d](https://github.com/valory-xyz/open-aea/pull/413/commits/8152d4d6056b257ffba5922377e67730dbcd7a5d)